Components/badge.css

```diff
@@ -0,0 +1,47 @@
+.badge {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 10px;
+}  
+  
+.badge-icon {
+    width: 50px;
+    height: 50px;
+    color: #333;
+    background-color: #ddd;
```

Comment:
Just one suggestion, use generic color variables

Reply:
Thanks , in next PR i will do use variables.